judo-runtime-core-jsl-itest/src/test/resources/model/MappedTransferPrimitivesModel/MappedTransferPrimitives.jsl

```diff
@@ -0,0 +1,194 @@
+type binary Binary(mime-types = ["text/plain"], max-file-size=1 GB);
+type string SmallString(min-size = 0, max-size = 128);
+
+enum Enum {
+    EnumA = 0;
+    EnumB = 1;
+    EnumC = 2;
+}
+
+entity EntityForMapping {
+        field Integer integerAttr;
+        field Scaled scaledAttr;
+        field SmallString stringAttr;
+        field PhoneNumber regexAttr;
+        field Boolean boolAttr;
+        field Date dateAttr;
+        field Timestamp timestampAttr;
+        field Time timeAttr;
+        field Enum enumAttr;
+        field Binary binaryAttr;
+}
+
+transfer TransferOptionalPrimitives (EntityForMapping mappedEntity) {
+    field Integer integerAttr maps mappedEntity.integerAttr;
+    field Scaled scaledAttr maps mappedEntity.scaledAttr;
+    field SmallString stringAttr maps mappedEntity.stringAttr;
+    field PhoneNumber regexAttr maps mappedEntity.regexAttr;
+    field Boolean boolAttr maps mappedEntity.boolAttr;
+    field Date dateAttr maps mappedEntity.dateAttr;
+    field Timestamp timestampAttr maps mappedEntity.timestampAttr;
+    field Time timeAttr maps mappedEntity.timeAttr;
+    field Enum enumAttr maps mappedEntity.enumAttr;
+    field Binary binaryAttr maps mappedEntity.binaryAttr;
+}
+
+entity EntityForMappingRequired {
+        field required Integer integerAttr ;
+        field required Scaled scaledAttr;
+        field required String stringAttr;
+        field required PhoneNumber regexAttr;
+        field required Boolean boolAttr;
+        field required Date dateAttr;
+        field required Timestamp timestampAttr;
+        field required Time timeAttr;
+        field required Enum enumAttr;
+        field required Binary binaryAttr;
+}
+
+transfer TransferRequiredPrimitives (EntityForMappingRequired mappedEntity) {
+    field required Integer integerAttr maps mappedEntity.integerAttr;
+    field required Scaled scaledAttr maps mappedEntity.scaledAttr;
+    field required String stringAttr maps mappedEntity.stringAttr;
+    field required PhoneNumber regexAttr maps mappedEntity.regexAttr;
+    field required Boolean boolAttr maps mappedEntity.boolAttr;
+    field required Date dateAttr maps mappedEntity.dateAttr;
+    field required Timestamp timestampAttr maps mappedEntity.timestampAttr;
+    field required Time timeAttr maps mappedEntity.timeAttr;
+    field required Enum enumAttr maps mappedEntity.enumAttr;
+    field required Binary binaryAttr maps mappedEntity.binaryAttr;
+}
+
+entity EntityWithPrimitiveDefaults {
+    field Integer integerAttr = 1;
+    field Scaled scaledAttr = 2.34;
+    field String stringAttr = "test";
+    field PhoneNumber regexAttr = "+36-1-123-123";
+    field Boolean boolAttr = true;
+    field Date dateAttr = `2022-07-11`;
+    field Timestamp timestampAttr = `2022-07-11T19:09:33Z`;
+    field Time timeAttr = `23:59:59`;
+    field Time shortTimeAttr = `23:59`;
+    field Binary binaryAttr; // There is no syntax for define default value for binary
```

Comment:
to define or for defining

Reply:
[6f85e57](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/165/commits/6f85e57884996176f05ef4554ae4669dee9e50a2)